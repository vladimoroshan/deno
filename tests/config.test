args: run --reload --config tests/config.tsconfig.json tests/config.ts
check_stderr: true
exit_code: 1
output: tests/config.ts.out
