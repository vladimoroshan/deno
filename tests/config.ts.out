[WILDCARD]Unsupported compiler options in "[WILDCARD]config.tsconfig.json"
  The following options were ignored:
    module, target
[WILDCARD]tests/config.ts:3:5 - error TS2532: Object is possibly 'undefined'.

3 if (map.get("bar").foo) {
      ~~~~~~~~~~~~~~

